### DISCUSSION TITLE
The original view of the message from TH + the conversation style from this addon

### DISCUSSION THREAD

**Discussion by maikcrew** [protz/thunderbird-conversations#1965] [Ideas]
I have a lot of templates that nicely modify the default display of the message, e.g. Quote colors, Tag Popup ... and of course they are not displayed in this template. Could it be possible to add the option to show the classic look even with the buttons with labels, only keeping the conversation look?

**Comment by Standard8** [comment]
If you mean that you're using the templates to modify the display of the header, then that would be too complex to maintain. That would require maintaining the same layout as Thunderbird which would not be practical.

If you are on about the body of the messages, that should generally work depending on how you are applying the templates. If you are using another add-on, then the issue is probably #1609
